lib/squid/plotter.rb

```diff
@@ -170,16 +170,20 @@ def item(point, color, w, fill, index, count)
     # symbol with the matching color. Labels are written from right to left.
     # @param
     def legend_item(label, x, color, options)
-      size, symbol_padding, entry_padding = 5, 3, 12
-      x -= @pdf.width_of(label, size: 7).ceil
-      @pdf.text_box label, options.merge(at: [x, @pdf.bounds.height])
+      size, symbol_padding, entry_padding = 5, 3, 28
+      x -= @pdf.width_of(label, size: 12).ceil
```

Comment:
semi magic numbers (i know, its from someone else's code, but we can Improve it :)